Assignment2/adrix1341.ttl

```diff
@@ -0,0 +1,13 @@
+@base <http://www.oeg-upm.net/Organization/>.
+@prefix pe: <http://www.ontologies.org/ontologies/People#>.
+@prefix pr: <http://www.oeg-upm.net/Properties#>.
+@prefix or: <http://www.oeg-upm.net/Organization#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>. 
+
+or:Class01 pr:includes or:Sensor029, or:Computer101.
+or:Sensor029 pr:hasMeasurement or:Measurement8401.
+or:Measurement8401 pr:hasTemperature "29" ^^xsd:integer;

```

Comment:
Ok, only thing is that normally you do not put a blank space between the symbols " and ^^, but I am accepting it.